### DISCUSSION TITLE
Nchain and StorCoin broken after upgrade to 7.1

### DISCUSSION THREAD

**Discussion by LeroyINC** [guydavis/machinaris#640] [General]
after performing an upgrade to 7.1 Chia and all forks are fine excpet for Nchain and Storcoin

when logging into the docker container for each fork and checkign the log.. i get the following:

NChain:
2022-04-04T16:59:00.873 wallet wallet                     : INFO     Reconnecting to peer {'host': '127.0.0.1', 'port': 58445}
2022-04-04T16:59:02.871 farmer farmer                     : INFO     Reconnecting to peer {'host': '127.0.0.1', 'port': 58445}

StorCoin:
2022-04-04T17:17:01.873 wallet wallet                     : INFO     Reconnecting to peer {'host': '127.0.0.1', 'port': 8668}
2022-04-04T17:17:02.871 farmer farmer                     : INFO     Reconnecting to peer {'host': '127.0.0.1', 'port': 8668}

these respective log entries juts repeat over and over.

**Comment by LeroyINC** [comment]
meant to add.. i know this is not a lot of info but was wondering if anyone had an idea off the top of their head.

**Comment by guydavis** [comment]
Hi, I have not seen this with these two blockchains in v0.7.1, however it is not uncommon for a particular blockchain to stop syncing.  Please try the following:

1. Restart the Machinaris-FORK container.  Or perhaps, just try a `stor start farmer -r` inside the container.
2. Add blockchain connections to get syncing moving.  I like the peers lists at https://alltheblocks.net/
3. If still not getting a "Syncing" status from `chia farm summary`, then try a [blockchain config reset](https://github.com/guydavis/machinaris/wiki/Troubleshooting#reset-blockchain-config).
4. If still not getting a "Syncing" status from `chia farm summary`, then try a [blockchain database reset](https://github.com/guydavis/machinaris/wiki/Troubleshooting#reset-blockchain-database).

Hope this helps!

**Comment by LeroyINC** [comment]
i stopped the nchain and storcoin docker.... then renamed the /root/.machinaris-nchain and stor folder to force it to recreate
this worked and it started to sync a new DB

then i tried to copy the old blockchain DB to the new container.. then i had the same issue.

must be something in the DB file that got corrupted or something.. so i am letting it sync again

**Comment by LeroyINC** [comment]
speaking of Database.. .does Machinaris upgrade the database to v2 when the fork supports it.. or do we need to manually do that?

**Reply by guydavis** [reply]
Manually:

- `chia db upgrade`
- `flax db upgrade` 

Only two I know of so far.

**Comment by LeroyINC** [comment]
thanks...

that's all I needed.. I have it working now... appreciate your help, and your time and energy on this project of yours
